### ISSUE TITLE
Still has issues with parsing of comments in Lua scripts.

### ISSUE THREAD

**Issue by SteelTitanium** [sirjuddington/SLADE#931]
Sorry for having to reopen this, but finally got around to testing it, as of dev build 2260844 that was uploaded to the site few days ago. The issue isn't fixed and it still has issues with certain comment types.

**Comment by sirjuddington** [comment]
Can you give an example of something that doesn't work?

**Comment by SteelTitanium** [comment]
Yes, below is a zip file containing an example WAD. When opened in SLADE, only the C style comment lines and block will be recognized, and will treat the standard Lua comment lines and blocks as if it was actual code. When both types of comment style should be treated as a comment, as defined in https://github.com/sirjuddington/SLADE/blob/master/dist/res/config/languages/srb2lua.txt.
[testscript.zip](https://github.com/sirjuddington/SLADE/files/2564379/testscript.zip)

**Comment by sirjuddington** [comment]
Oh I see, this is because the srb2lua language definition has two different block comment styles defined (`/*...*/` and `--[[...]]`). Currently SLADE only supports one begin/end token for comments per language.

**Comment by crisisinaptica** [comment]
Should be easy to implement, I'll check this one later.
